src/listener.ts

```diff
@@ -200,25 +203,31 @@ class Listener extends EventEmitter implements InterfaceListener {
             },
             async () => {
               this.udpSocket.removeListener('error', reject)
-              try {
-                this.externalAddress = await getExternalIp(this.stunServers, this.udpSocket)
-              } catch (err) {
-                error(`Unable to fetch external address using STUN. Error was: ${err}`)
-              }
+
+              this.externalAddress = await getExternalIp(this.stunServers, this.udpSocket)
+
+              // } catch (err) {
+              //   error(`Unable to fetch external address using STUN. ${err}`)
+              // }
 
               resolve(this.udpSocket.address().port)
             }
           )
         } catch (err) {
-          error(`Could bind UDP socket. Error was: ${err.message}`)
+          error(`Could not bind to UDP socket. ${err.message}`)
           reject()
         }
       })
 
     if (options.port == 0 || options.port == null) {
-      await listenTCP().then((port) => listenUDP(port))
+      const tcpPort = await listenTCP()
+      listenUDP(tcpPort)
     } else {
-      await Promise.all([listenTCP(), listenUDP()])
+      await Promise.all([
+        // prettier-ignore
```

Comment:
Remove prettier-ignore

Reply:
I try to avoid `await Promise.all([listenTCP(), listenUDP()])` syntactically to make sure I don't lose any async operations. Without prettier-ignore, this gets formatted by Prettier